fixture-monkey-kotlin/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-reflect:${Versions.KOTLIN}")
 
     testImplementation(project(":fixture-monkey-jackson"))
+    testImplementation(project(":fixture-monkey-javax-validation"))
```

Comment:
Is this dependency necessary?

Reply:
In the ExpressionGeneratorJavaTestSpecs, javax validation was previously being used. 

Even on the main branch, compile was not successful in my local environment, the exact reason is unclear.

After changing modules, [compile fails in the absence of the dependency](https://github.com/naver/fixture-monkey/actions/runs/8321197148/job/22767082469), so it necessary in the current situation.